packages/console/src/pages/Applications/components/QuickStartGuide/index.tsx

```diff
@@ -0,0 +1,139 @@
+import classNames from 'classnames';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import highFive from '@/assets/images/high-five.svg';
+import tada from '@/assets/images/tada.svg';
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import Spacer from '@/components/Spacer';
+import Close from '@/icons/Close';
+import { SupportedJavascriptFrameworks } from '@/types/applications';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  appName: string;
+  onClose: () => void;
+};
+
+const QuickStartGuide = ({ appName, onClose }: Props) => {
+  const [isLibrarySelectorFolded, setIsLibrarySelectorFolded] = useState(false);
+  const [libraryName, setLibraryName] = useState<string>(SupportedJavascriptFrameworks.Angular);
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const onClickFetchSampleProject = () => {
+    window.open(
+      `https://github.com/logto-io/js/tree/master/packages/${libraryName.toLowerCase()}-sample`,
+      '_blank'
+    );
+  };
+
+  const librarySelector = useMemo(
+    () => (
+      <Card className={classNames(styles.card, styles.selector)}>
+        <img src={highFive} alt="success" />
+        <CardTitle
+          title="applications.quick_start.title"
+          subtitle="applications.quick_start.subtitle"
+        />
+        <RadioGroup
+          name="frameworkName"
+          value={libraryName}
+          onChange={(value) => {
+            setLibraryName(value);
+          }}
+        >
+          {Object.values(SupportedJavascriptFrameworks).map((framework) => (
+            <Radio key={framework} className={styles.radio} title={framework} value={framework} />
+          ))}
+        </RadioGroup>
+        <div className={styles.buttonWrapper}>
+          <Spacer />
+          <Button
+            type="primary"
+            title="general.next"
+            onClick={() => {
+              setIsLibrarySelectorFolded(true);
+            }}
+          />
+        </div>
+      </Card>
+    ),
+    [libraryName]
+  );
+
+  const librarySelectorFolded = useMemo(
+    () => (
+      <div className={classNames(styles.card, styles.selector, styles.folded)}>
+        <img src={tada} alt="Tada!" />
+        <span>
+          {t('applications.quick_start.description_by_framework', { library: libraryName })}
+        </span>
+      </div>
```

Comment:
Per design missing a quick library selector here?

Reply:
Yes, that one requires a dropdown component, which currently we don't have. Have created [linear issue](https://linear.app/silverhand/issue/LOG-1923/single-selection-dropdown) to track this.

Reply:
ActionMenu?

Reply:
It has a demo in "delete user"

Reply:
ActionMenu UI does not match the design here, I think